src/components/card/types.ts

```diff
@@ -0,0 +1,9 @@
+import { ReactNode } from 'react'
+
+interface CardProps {
+  children: ReactNode
+  className?: string
+  elevation?: 'base' | 'low' | 'mid' | 'high'
+}
+
+export type { CardProps }
```

Comment:
This could be changed to the following without any impact, and I think aligns with what we do elsewhere:
```suggestion
export interface CardProps {
  children: ReactNode
  className?: string
  elevation?: 'base' | 'low' | 'mid' | 'high'
}
```

Reply:
Ah, okay! I think I went with `export type` because of something about cyclical exports? Not sure if I applied that correctly, but I remember you mentioning it during pairing some time ago. Do you remember what we were talking about?

Reply:
Hmm, I do not 😅 

I don't think cyclical imports/exports would be a problem here, I think these two are functionally equivalent. cyclical imports could come into play if we were, say, importing something from the index file in the types file, and also importing something from the types file in the index file.

Reply:
That makes sense, thanks for elaborating! The update has been made in https://github.com/hashicorp/dev-portal/pull/83/commits/f49491e9c9b3d5445131c4f1252d0b3fbf50af7a. 😊